source/extensions/filters/network/thrift_proxy/conn_manager.cc

```diff
@@ -404,6 +412,13 @@ void ConnectionManager::ActiveRpc::finalizeRequest() {
 
   parent_.stats_.request_.inc();
 
+  parent_.accumulated_requests_++;
+
+  if (parent_.accumulated_requests_ >= parent_.config_.maxRequestsPerConnection()) {
+    parent_.read_callbacks_->connection().readDisable(true);
+    parent_.requests_overflow_ = true;
```

Comment:
can we add a stat for this and increment it here?

Reply:
Get.